src/jukebox/run_rpc_tool.py

```diff
@@ -309,6 +309,42 @@ def main(scr):
             scr.addstr(f"\n:: Response =\n{response}\n\n")
 
 
+def runcmd(cmd):
```

Comment:
The name of the file and the new functionality sound a bit strange now.

Should `run_rpc_tool.py` be split into two parts, a cli client and the interactive tool? 
Like `pbc.c` the cli client part could stay in the "cli_client" folder, to be executed from external.
The `run_rpc_tool.py` still gives the interactive tool and runs the commands with the cli client, so no duplications are needed?

Dont't know though if the script in cli_client would work with the python imports.

Reply:
My intention was to do both in one tool, because I found it confusing when I tried to figure out which tool I needed for my use case.

But I think it's also an approach to have one tool each for each use case. So then improving the docs would be necessary.

Reply:
I would like the approach of one tool for each use case, but doens't have be implemented right away.